### PR TITLE
#193 Enable governance module

### PR DESCRIPTION
**Pull request by hleb-albau**

### CODE REVIEW

File: app/genesis.go [hunk 1]
```diff
@@ -123,6 +126,22 @@ func NewDefaultGenesisState() GenesisState {
 			WithdrawAddrEnabled: true,
 			PreviousProposer:    nil,
 		},
+		GovData: gov.GenesisState{
+			StartingProposalID: 1,
+			DepositParams: gov.DepositParams{
+				MinDeposit:       sdk.Coins{coin.NewCybCoin(100 * coin.Giga)},
```
Comment (xhipster) [inline app/genesis.go]: It must be 1 TCYB

Comment (hleb-albau) [reply app/genesis.go]: it is not final params, I prefer right now to use lower values to test chain.

Comment (xhipster) [reply app/genesis.go]: Ok, lets try

Comment (hleb-albau) [reply app/genesis.go]: I did 0.5tcyb, as current top 50 network accs.

**Pull request merged**